drop gcp-project-type from presubmits

kannon92 opened the pull request:
Moving these periodics to presubmits.  I notice that in the presubmit file we don't have gcp-project-type.  

Seeing failures in https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/120679/pull-crio-cgroupv1-node-e2e-eviction/1704873947100614656/

aojea commented:
what are those failures?

kannon92 commented:
> what are those failures?

I was getting 
```
{  --provider=gce boskos failed to acquire project: resources not found}
```

```
W0921 15:12:09.378] (for local runs please set --gcp-project to your dev project)
W0921 15:12:09.378] 2023/09/21 15:07:09 main.go:758: provider gce, will acquire project type node-e2e-project from boskos
W0921 15:12:09.378] 2023/09/21 15:12:09 process.go:96: Saved XML output to /workspace/_artifacts/junit_runner.xml.
W0921 15:12:09.746] 2023/09/21 15:12:09 process.go:153: Running: bash -c . hack/lib/version.sh && KUBE_ROOT=. kube::version::get_version_vars && echo "${KUBE_GIT_VERSION-}"
W0921 15:12:09.746] 2023/09/21 15:12:09 process.go:155: Step 'bash -c . hack/lib/version.sh && KUBE_ROOT=. kube::version::get_version_vars && echo "${KUBE_GIT_VERSION-}"' finished in 368.803239ms
W0921 15:12:09.749] 2023/09/21 15:12:09 main.go:328: Something went wrong: failed to prepare test environment: --provider=gce boskos failed to acquire project: resources not found
W0921 15:12:09.750] Traceback (most recent call last):
W0921 15:12:09.750]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 723, in <module>
W0921 15:12:09.750]     main(parse_args())
W0921 15:12:09.751]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 569, in main
W0921 15:12:09.751]     mode.start(runner_args)
W0921 15:12:09.751]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 228, in start
W0921 15:12:09.751]     check_env(env, self.command, *args)
W0921 15:12:09.751]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 111, in check_env
W0921 15:12:09.751]     subprocess.check_call(cmd, env=env)
W0921 15:12:09.762]   File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
W0921 15:12:09.762]     raise CalledProcessError(retcode, cmd)
W0921 15:12:09.762] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--deployment=node', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--gcp-project-type=node-e2e-project', '--gcp-zone=us-west1-b', '--node-test-args=--container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\\"name\\": \\"crio.log\\", \\"journalctl\\": [\\"-u\\", \\"crio\\"]}"', '--node-tests=true', '--test_args=--nodes=1 --focus="\\[NodeFeature:Eviction\\]"', '--timeout=300m', '--node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv1.yaml')' returned non-zero exit status 1.
E0921 15:12:09.762] Command failed
I0921 15:12:09.763] process 524 exited with code 1 after 5.0m
E0921 15:12:09.763] FAIL: pull-crio-cgroupv1-node-e2e-eviction
I0921 15:12:09.763] Call:  gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
W0921 15:12:10.601] Activated service account credentials for: [prow-build@k8s-infra-prow-build.iam.gserviceaccount.com]
I0921 15:12:10.717] process 602 exited with code 0 after 0.0m
I0921 15:12:10.717] Call:  gcloud config get-value account
I0921 15:12:11.560] process 612 exited with code 0 after 0.0m
I0921 15:12:11.560] Will upload results to gs://kubernetes-jenkins/pr-logs using prow-build@k8s-infra-prow-build.iam.gserviceaccount.com
I0921 15:12:11.560] Upload result and artifacts...
I0921 15:12:11.560] Gubernator results at https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/120679/pull-crio-cgroupv1-node-e2e-eviction/1704873947100614656
I0921 15:12:11.561] Call:  gsutil ls gs://kubernetes-jenkins/pr-logs/pull/120679/pull-crio-cgroupv1-node-e2e-eviction/1704873947100614656/artifacts
W0921 15:12:12.965] CommandException: One or more URLs matched no objects.
E0921 15:12:13.[182](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/120679/pull-crio-cgroupv1-node-e2e-eviction/1704873947100614656/#1:build-log.txt%3A182)] Command failed
```

I see logs mention the node-e2e-project but we don't have that project in any of our other presubmits.

aojea commented:
/assign @BenTheElder 

it may be related indeed, but not completely familiar with this setup

kannon92 commented:
/assign @SergeyKanzhelev

upodroid reviewed (commented):
Can you move these jobs to the community cluster?

You can do this by adding the field `cluster: k8s-infra-prow-build` to the job

xref https://github.com/kubernetes/test-infra/issues/29722

kannon92 commented:
> Can you move these jobs to the community cluster?
> 
> You can do this by adding the field `cluster: k8s-infra-prow-build` to the job
> 
> xref #29722

I looked over and I was only missing one of them.  

Converted the periodic crio jobs to use the community cluster. https://github.com/kubernetes/test-infra/pull/30793

kannon92 commented:
/retest

SergeyKanzhelev commented:
does this needs to be rebased now? I am not sure I understand all the differences between projects, but this PR seems incomplete with one job adding cluster and others don't

kannon92 commented:
> does this needs to be rebased now? I am not sure I understand all the differences between projects, but this PR seems incomplete with one job adding cluster and others don't

I rebased earlier today.  

I only saw one job in the presubmits that was missing the cluster so I added that.  

And the I was told that the gcp-project-type may not be required for the presubmits (#sig-testing help mostly).

kannon92 commented on config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:
```diff
@@ -1134,6 +1134,7 @@ presubmits:
         - '--test-args=--container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u ...
         - --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv2-k8s-infra-prow-build-kubetest2.yaml
   - name: pull-crio-cgroupv1-node-e2e-features
+    cluster: k8s-infra-prow-build
```
This is the only job that was missing a cluster tag.  All other presubmits have a cluster so they have already been migrated to the community cluster.

kannon92 commented on config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:
```diff
@@ -1161,7 +1162,6 @@ presubmits:
         args:
         - --deployment=node
         - --env=KUBE_SSH_USER=core
-        - --gcp-project-type=node-e2e-project
```
Not entirely sure why but it seems that presubmits don't require this project-type.

SergeyKanzhelev reviewed (approved):
/lgtm
/approve
